pkg/scheduler/reconciliation/filter.go

```diff
@@ -157,6 +157,27 @@ func (ws *WithSchedulingID) FilterByInstance(i *model.ReconciliationEntity) *mod
 	return nil
 }
 
+type WithNotSchedulingID struct {
+	SchedulingID string
+}
+
+func (wns *WithNotSchedulingID) FilterByQuery(q *db.Select) error {
+	column, err := columnName(q, "SchedulingID")
+	if err != nil {
+		return err
+	}
+
+	q.WhereRaw(fmt.Sprintf("%s<>$%d", column, q.NextPlaceholderCount()), wns.SchedulingID)
```

Comment:
I think this is not strictly necessary, but could we make use of the existing positional argument support instead of building the entire query on demand? We still make use of the SQL Injection Regex so I dont think its a big problem but I think here we could use it.

Reply:
That would have been good, but there is no support for an operator other than "=" in Where() at the moment.